src/ja/user/cli.py

```diff
@@ -1,19 +1,244 @@
+        parser.add_argument("--passphrase", "--pp", type=str, default=passphrase,
+                            help="The passphrase to use for decrypting private keys.")
+
+        subparsers = parser.add_subparsers(dest="command")
+        # parser for add command.
+        parser_add = subparsers.add_parser("add")
+        parser_add.add_argument("--config", "-c", help="Config file path for adding a job.")
+        parser_add.add_argument("--label", "-l", help="Label for the job.")
+        parser_add.add_argument("--source", "-s", "--path", help="Path of the source file containing the job.")
+        parser_add.add_argument("--priority", "-p", choices="low medium high urgent 0 1 2 3".split())
+        parser_add.add_argument("--threads", "-t", type=int, help="Assign a number of cpu threads for your job.")
+        parser_add.add_argument("--memory", "-m", type=int, help="Assign a specific amount of memory to your job")
+        parser_add.add_argument("--preemptible", "--pe", action="store_true")
+        parser_add.add_argument("--blocking", "--bl", action="store_true")
+        parser_add.add_argument("--mount", nargs="+", type=str)
+        parser_add.add_argument("--email", "-e", required=email is None, default=email)
+        parser_add.add_argument("--special-resources", "--sr", nargs="+",
+                                help="Add the required special resources for a job.")
+        parser_add.add_argument("--owner", default=-1, type=int, help=argparse.SUPPRESS)
+
+        # parser for query command
+        parser_query = subparsers.add_parser("query")
+        parser_query.add_argument("--uid", "-u", nargs="+", help="Job uid(s) to filter query results by.")
+        parser_query.add_argument("--label", "-l", nargs="+", help="Job label(s) to filter query results by.")
+        parser_query.add_argument("--owner", "-o", nargs="+", help="Job owners to filter query results by.")
+        parser_query.add_argument("--priority", "-p",
+                                  nargs="+",
+                                  choices="low medium high urgent 0 1 2 3".split(),
+                                  help="Job priorities to filter query results by.")
+        parser_query.add_argument("--status",
+                                  nargs="+",
+                                  choices="new queued running paused done killed crashed 0 1 2 3 4 5 6".split(),
+                                  help="Job status(es) to filter query results by.")
+        parser_query.add_argument("--preemptible", "--is-preemptible", "--ip",
+                                  dest="preemptible",
+                                  choices="false true t f".split(),
+                                  help="Get job(s) that can be preempted by other jobs.")
+        parser_query.add_argument("--special-resources", "--sr", help="Filter by the special resources of a job. \
```

Comment:
What do the lists separated by '.' represent? Should be clarified by the help message.